### ISSUE TITLE
[comma-dangle] No dangling comma required in multi-line interface with "always-multiline"

### ISSUE THREAD

**Issue by FichteFoll** [typescript-eslint/typescript-eslint#4903]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/comma-dangle": [
      "error",
      "always-multiline"
    ]
  }
}
```

```TS
interface Test {
    hi: string
}
```

https://typescript-eslint.io/play/#ts=4.6.2&sourceType=module&code=JYOwLgpgTgZghgYwgAgCoQM5mQbwFDKHIAWwAXMllKAOZ4C+QA&rules=N4IgAgLgngDgpgZwMYCcCWMIFpEBs0B2EA9EgPYC2FAhlgCbUEDmucIAXANohwopkoQAGhDVcAd2pQEWCgFdcENPgJsAugF8gA&tsConfig=N4XyA

**Expected Result**

The missing dangling comma should be annotated (and fixed when run with `--fix`).

**Actual Result**

The code passes the check.

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.22.0` |
| `@typescript-eslint/parser`        | `5.22.0` |
| `TypeScript`                       | `4.6.4` & `4.6.2` (playground) |
| `ESLint`                           | `8.14.0` & `8.7.0` (playground) |
| `node`                             | `16.14.2` |

**Comment by bradzacher** [comment]
Use our member-delimiter-style rule instead.

https://typescript-eslint.io/rules/member-delimiter-style
